mocks/organisasjon-mock/src/main/java/no/nav/tjeneste/virksomhet/organisasjon/rs/OrganisasjonRSV1Mock.java

```diff
@@ -43,23 +42,46 @@
             @ApiImplicitParam(name = "inkluderHierarki", dataType = "string", paramType = "query"),
             @ApiImplicitParam(name = "inkluderHistorikk", dataType = "string", paramType = "query")
     })
-    public Response hentOrganisasjon(@PathParam("orgnummer") String orgnummer,
+    public OrganisasjonEReg hentOrganisasjon(@PathParam("orgnummer") String orgnummer,
+                                             @Context HttpHeaders httpHeaders,
+                                             @Context UriInfo uriInfo) {
+        if (orgnummer != null) {
+            LOG.info("EREG REST {}", orgnummer);
+            Optional<OrganisasjonModell> organisasjonModell = scenarioRepository.getOrganisasjon(orgnummer);
+            if (organisasjonModell.isPresent()) {
+                OrganisasjonModell modell = organisasjonModell.get();
+                return new OrganisasjonEReg(modell);
+            } else {
+                return null;
+            }
+        } else {
+            throw new IllegalArgumentException("Orgnummer ikke angitt");
+        }
+    }
+
+    @SuppressWarnings("unused")
+    @GET
+    @Path("/{orgnummer}")
+    @ApiOperation(value = "Henter adresse informasjon for et organisasjonsnummer")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "inkluderHierarki", dataType = "string", paramType = "query"),
+            @ApiImplicitParam(name = "inkluderHistorikk", dataType = "string", paramType = "query")
+    })
+    public OrganisasjonAdresse hentOrganisasjonAdresse(@PathParam("orgnummer") String orgnummer,
```

Comment:
Hva er grunnen til at du ikke bruker Response her? Slik som tidligere.

Reply:
Fordi det er to forskjellige metoder som returnerer to forskjellige objekter...? En for OrganisasjonEReg og en for OrganisasjonAdresse. Jeg fikk forklart at det er slik man kan få samme rest tjeneste til å returnere forskjellige svar ettersom hva requestern ønsker... Er det feil?